files/jenkins-slave.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+JENKINS_CONFIG=/etc/sysconfig/jenkins-slave
+
+# Read config
+[ -f "$JENKINS_CONFIG" ] && . "$JENKINS_CONFIG"
+
+if [ -x /sbin/runuser ] ; then
+    RUNUSER=runuser
+else
+    RUNUSER=su
```

Comment:
The wrapper was created so settings can still be stored in the defaults/sysconfig file and used from it.